Traceback (most recent call last):
  File "C:\Users\Dmitriy Kornilov\AppData\Roaming\Python\Python310\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\Dmitriy Kornilov\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\Dmitriy Kornilov\AppData\Roaming\Python\Python310\site-packages\jupyter_core\utils\__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
  File "C:\Program Files\Python310\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "C:\Users\Dmitriy Kornilov\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "C:\Users\Dmitriy Kornilov\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\Dmitriy Kornilov\AppData\Roaming\Python\Python310\site-packages\nbclient\client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import pandas as pd
import os

from nilearn import plotting
%matplotlib inline


from nipype.interfaces.spm import Level1Design, EstimateModel, EstimateContrast
from nipype.algorithms.modelgen import SpecifySPMModel
from nipype.interfaces.utility import Function, IdentityInterface
from nipype.interfaces.io import SelectFiles, DataSink
from nipype import Workflow, Node
from utils import list_files
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [1;32mIn[5], line 8[0m
[0;32m      4[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnilearn[39;00m [38;5;28;01mimport[39;00m plotting
[0;32m      5[0m get_ipython()[38;5;241m.[39mrun_line_magic([38;5;124m'[39m[38;5;124mmatplotlib[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124minline[39m[38;5;124m'[39m)
[1;32m----> 8[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnipype[39;00m[38;5;21;01m.[39;00m[38;5;21;01minterfaces[39;00m[38;5;21;01m.[39;00m[38;5;21;01mspm[39;00m [38;5;28;01mimport[39;00m Level1Design, EstimateModel, EstimateContrast
[0;32m      9[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnipype[39;00m[38;5;21;01m.[39;00m[38;5;21;01malgorithms[39;00m[38;5;21;01m.[39;00m[38;5;21;01mmodelgen[39;00m [38;5;28;01mimport[39;00m SpecifySPMModel
[0;32m     10[0m [38;5;28;01mfrom[39;00m [38;5;21;01mnipype[39;00m[38;5;21;01m.[39;00m[38;5;21;01minterfaces[39;00m[38;5;21;01m.[39;00m[38;5;21;01mutility[39;00m [38;5;28;01mimport[39;00m Function, IdentityInterface

[1;31mModuleNotFoundError[0m: No module named 'nipype'

